pkg/minikube/bootstrapper/localkube/privileges.go

```diff
@@ -0,0 +1,37 @@
+package localkube
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	rbacv1beta1 "k8s.io/client-go/pkg/apis/rbac/v1beta1"
+	"k8s.io/minikube/pkg/minikube/service"
+)
+
+func elevateKubeSystemPrivileges() error {
+	k8s := service.K8sClientGetter{}
+	client, err := k8s.GetRBACV1Beta1Client()
+	if err != nil {
+		return err
+	}
+	clusterRoleBinding := &rbacv1beta1.ClusterRoleBinding{
+		ObjectMeta: v1.ObjectMeta{
+			Name: "minikube-rbac",
+		},
+		Subjects: []rbacv1beta1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "default",
+				Namespace: "kube-system",
+			},
+		},
+		RoleRef: rbacv1beta1.RoleRef{
+			Kind: "ClusterRole",
+			Name: "cluster-admin",
```

Comment:
Correct me if i'm wrong, but I believe most RBAC-enabled clusters are not setup to give the `kube-system:default` service account `cluster-admin` privileges. Is it maybe safer to grant these privileges in the addons and services that need them, or perhaps create a new `minikube-default` service account to use in pods that would only exist in minikube?